token/cli/src/main.rs

```diff
@@ -7507,10 +7474,22 @@ mod tests {
                 "--program-id",
                 &program_id.to_string(),
                 "--metadata-address",
-                metadata_address.to_string().as_str()
+                metadata_address.to_string().as_str(),
             ],
         )
-        .await
-        .unwrap();
+        .await;
+
+        let value: serde_json::Value = serde_json::from_str(&result.unwrap()).unwrap();
+        let mint = Pubkey::from_str(value["commandOutput"]["address"].as_str().unwrap()).unwrap();
+        let account = config.rpc_client.get_account(&mint).await.unwrap();
+        let mint_state = StateWithExtensionsOwned::<Mint>::unpack(account.data).unwrap();
+
+        let extension = mint_state.get_extension::<MetadataPointer>().unwrap();
+
+        assert!(mint_state.get_extension::<MetadataPointer>().is_ok());
```

Comment:
nit: you can remove this line since you already unwrapped on the same call one line before

Reply:
Sorry, I meant this exact line, not the whitespace that was there before